ember-headless-form/src/components/-private/control/checkbox.ts

```diff
@@ -0,0 +1,19 @@
+import Component from '@glimmer/component';
+import { assert } from '@ember/debug';
+import { action } from '@ember/object';
+
+export interface HeadlessFormControlCheckboxComponentSignature {
+  Element: HTMLInputElement;
+  Args: {
+    value: boolean;
+    fieldId: string;
+    setValue: (value: boolean) => void;
+  };
+}
+
+export default class HeadlessFormControlCheckboxComponent extends Component<HeadlessFormControlCheckboxComponentSignature> {
```

Comment:
are these names from the generator? they're very long 😅

Reply:
No, they are a product of my own creativity! 🤷‍♂️ 😆

Seriously though, even when verbose, they follow the logical (folder) structure, which helps me to keep them consistent...